Functions/Utilities/New-RsConfigurationSettingObject.ps1

```diff
@@ -4,53 +4,108 @@
 function New-RsConfigurationSettingObject
 {
     <#
-    .SYNOPSIS
-        This script creates a new WMI Object that is connected to Reporting Services WMI Provider.
-
-    .DESCRIPTION
-        This script creates a new WMI Object that is connected to Reporting Services WMI Provider. 
-
-    .PARAMETER SqlServerInstance (optional)
-        Specify the name of the SQL Server Reporting Services Instance.
-    
-    .PARAMETER SqlServerVersion (optional)
-        Specify the version of the SQL Server Reporting Services Instance. 13 for SQL Server 2016, 12 for SQL Server 2014, 11 for SQL Server 2012
-
-    .EXAMPLE 
-        New-RsConfigurationSettingObject 
-        Description
-        -----------
-        This command will retrieve and return WMI Object associated to the default instance (MSSQLSERVER) of SQL Server 2016 Reporting Services.
-
-    .EXAMPLE 
-        New-RsConfigurationSettingObject -SqlServerInstance 'SQL2016' 
-        Description
-        -----------
-        This command will retrieve and return WMI Object associated to the named instance (SQL2016) of SQL Server 2016 Reporting Services.
-
-    .EXAMPLE 
-        New-RsConfigurationSettingObject -SqlServerVersion '11' 
-        Description
-        -----------
-        This command will retrieve and return WMI Object associated to the default instance (MSSQLSERVER) of SQL Server 2012 Reporting Services.
-
-    .EXAMPLE 
-        New-RsConfigurationSettingObject -SqlServerInstance 'SQL2012' -SqlServerVersion '11' 
-        Description
-        -----------
-        This command will retrieve and return WMI Object associated to the named instance (SQL2012) of SQL Server 2012 Reporting Services.
+        .SYNOPSIS
+            This script creates a new WMI Object that is connected to Reporting Services WMI Provider.
+        
+        .DESCRIPTION
+            This script creates a new WMI Object that is connected to Reporting Services WMI Provider.
+        
+        .PARAMETER ReportServerInstance
+            Specify the name of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ReportServerVersion
+            Specify the version of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ComputerName
```

Comment:
By default, you are not able to connect to Reporting Services WMI provider on a remote machine without doing additional work. My only concern with introducing a parameter like this is that users will expect this command to work without realizing that the reason why it is failing is due to not having done this additional work. (Configure a Report Server for Remote Administration - https://msdn.microsoft.com/en-us/library/ms365170.aspx)